### DISCUSSION TITLE
Is `useStorage` really reactive?

### DISCUSSION THREAD

**Discussion by bartenra** [vueuse/vueuse#754] [Q&A]
When I use `useStorage`

```ts
const storage = useStorage('hello', 'how are you', undefined, { listenToStorageChanges: true });
```

And then call `localStorage.setItem('hello', 'world');` in the console, it doesn't update.

Is this expected behavior?

**Comment by bartenra** [comment]
Hmm when I call it in another component, it reactively updates.

I wonder why it doesn't work when I change localStorage through the console.

**Comment by antfu** [comment]
Try `await nextTick()` before accessing the value?

**Comment by JensErikChristiansen** [comment]
A little late here, but I believe you don't directly use localStorage; you use your `storage` variable.

```javascript
storage.value = 'doing fine!'
```

your browser's storage will update, as well as anything else listening to your `storage` var; that's the purpose of `useStorage`: to abstract away `localStorage`!
